spec/rubocop/cop/bundler/ordered_gems_spec.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 describe RuboCop::Cop::Bundler::OrderedGems, :config do
+  let(:cop_config) { { 'EnforcedStyle' => enforced_style, 'Include' => nil } }
```

Comment:
There are lots of fails without it. This solution is used in other specs as well, e.g. [here](https://github.com/bbatsov/rubocop/blob/master/spec/rubocop/cop/rails/not_null_column_spec.rb#L5).